package.json

```diff
@@ -14,7 +14,7 @@
     "format": "prettier --write \"src/**/*.ts\"",
     "lint": "eslint . --ext .ts",
     "lintAndFix": "eslint . --ext .ts --fix",
-    "prepare": "npm run build",
+    "prepare": "npm run rollup",
```

Comment:
this is wrong I think ? You do want to clean any old publication, do a build with `tsc` and then have rollup run on top of that. For rollup to run, you also need to add `-c` to tell it to look at the config file for what to do.

Finally, I think that this shouldn't be added to `prepare` but instead to `build`, as it's a build step?

Reply:
While I agree that we should move it to the `build` script, if you look closely the `build` script runs the `clean` script which delete the `lib`directory and `bundle.js`